starter/source/elements/shell/coqueba/cnepsini.F

```diff
@@ -165,7 +164,7 @@ SUBROUTINE CNEPSINI(ELBUF_STR,MAT_PARAM,
 c-----------------
       IF (MTN==58) THEN
         NITRS0 = NITRS
-        NITRS = MAX(IPLA_D*100,NITRS0)
+!        NITRS = MAX(IPLA_D*100,NITRS0)
```

Comment:
Hi @qiangRadioss 

Maybe I missed something , but , think that NITRS computation is commented , but used just after ( line 169)

Reply:
Hi Sebastien,
NITRS is defined in an include file and was initialized before with the input, NITRS0 was used to restore the value of NITRS with a hide option in case of law58, now i've removed this hide option (it should be) w/o change the value of NITRS.

thanks.
Qiang.